Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM python: latest
 
-COPY main.py binanceinfo.py Telegramsender.py .gitignore
+COPY main.py binanceinfo.py Telegramsender.py
```

Comment:
Looks like incorrect. To copy many files into image you need to pass list at the end of which need to put destination